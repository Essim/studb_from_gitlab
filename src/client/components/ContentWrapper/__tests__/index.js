import React from 'react';
import {ContentWrapper} from '../';
import { shallowToJson } from 'enzyme-to-json';

const mockChild = <div key={'mockKey'}>mock</div>;
const mockChildren= [mockChild];

describe('ContentWrapper', () => {
    it('should render correctly', () => {
        const output = shallow(<ContentWrapper children={mockChildren}/>);
        expect(shallowToJson(output)).toMatchSnapshot();
    });
});
