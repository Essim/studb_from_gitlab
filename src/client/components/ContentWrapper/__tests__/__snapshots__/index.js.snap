// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentWrapper should render correctly 1`] = `
<div
  className="content-wrapper"
>
  <section
    className="content"
  >
    <div
      key="mockKey"
    >
      mock
    </div>
  </section>
</div>
`;
